UIComponents/Tests/UIComponentsTests/FontExtensionTests.swift

```diff
@@ -0,0 +1,204 @@
+import XCTest
+
+final class FontExtensionTests: XCTestCase {
+    // MARK: LARGE TITLE
+    func test_largeTitle() {
```

Comment:
Can you add some expectations to the test name, please? i.e `test_largeTitle_hasCorrectSize()`